spec/02-integration/04-admin_api/14-tags_spec.lua

```diff
@@ -73,6 +73,16 @@ describe("Admin API - tags", function()
         end
       end)
 
+      it("filter by empty tag", function()
```

Comment:
Is there a way to test with empty string too?

Reply:
Oh good catch. Turns out that sending explicit empty string is fine. I've added a test for that to make sure we don't return 500 response in that case.